rfcs/2024-03-28-MQTT_Default_Presets

```diff
@@ -0,0 +1,61 @@
+
+    Start Date: 2024-03-28
+    RFC PR: Meshtastic/rfcs#0000
+    Affected Components: (e.g., Firmware, all clients)
+#Summary
+Currently, many people turn on the MQTT settings and unknowingly flood the local mesh with traffic while also uploading other people's traffic to the internet. My proposal is that a second set of modem presets are used for this. This will force a segregation between the people that want to use the online features and those that don’t by default. RF packets on this new setting would still be meshed to other users with this same setting but with these users only, thereby not piggybacking on the default long/fast mesh.
+
+#Motivation
+[Why are we doing this? What problem does it solve or what use cases does it support? Discuss the benefits for the different components of the Meshtastic ecosystem, including end-users, developers, and core maintainers.]
+A single user can currently flood the local public network with out-of-region traffic that gets propagated through the mesh, cluttering up maps and reducing limited bandwidth on long/fast. By focusing on modem presets tailored to higher traffic (and shorter range) this new mesh can more easily handle the influx of traffic. The shorter range will be compensated for by the assumption that there will be more like-minded people in each region passing on the traffic to MQTT.
+
+#Ecosystem Impact
+[Explain how this change will affect the various components of the Meshtastic ecosystem, including firmware, phone, desktop, and web applications. Address any coordination required among these components.]
+The two separate "default" meshes in each region will have incompatible modem settings. 
+Private channel settings won’t be affected.
```

Comment:
The modem preset applies to all channels, so if someone has LongFast with MQTT downlink enabled, also private channels will be affected.

Reply:
Can we do this only if they are using the public key?  Nobody really uses the other presets.